### DISCUSSION TITLE
Seems like I cannot open files from discussion

### DISCUSSION THREAD

**Discussion by ElricleNecro** [harrisoncramer/gitlab.nvim#221] [Q&A] [answered]
I have a merge request with multiple discussions in progress. From what I understood, when I do `o` on a node in the discussion window, it should bring me to the location in a file where the comment was made, right?

If yes, I'm getting the following error:
```
Error executing lua: Vim:E5300: Expected a Number or a String
stack traceback:
	[C]: in function 'bufnr'
	...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:456: in function 'jump_to_file'
	...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:832: in function <...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:830>
```

Using `require("gitlab").move_to_discussion_tree_from_diagnostic()` on a line with a comment marker does bring me to the node in the discussion view, but `o` on the node it sent me does not bring me back there, it just raise the above error.

Am I missing something?

**Comment by ElricleNecro** [comment]
I think this is actually a bug, or something that changed in a dependency. In the given file:
```lua
  local bufnr = vim.fn.bufnr(root_node.filename)
```
should actually be:
```lua
  local bufnr = vim.fn.bufnr(root_node.file_name)
```
(same for the following usage of `root_node.filename`).

After fixing this come another problem:
```
E5108: Error executing lua: ...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:467: Cursor position outside buffer
stack traceback:
	[C]: in function 'nvim_win_set_cursor'
	...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:467: in function 'jump_to_file'
	...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:834: in function <...lazy/gitlab.nvim/lua/gitlab/actions/discussions/init.lua:832>
```
which is link to the behaviour of `get_new_line` that seems to be trying to parse a string (`root_node.range.start.line_code`) when the line number (`root_node.range.start.new_line`) is already available and correct (for my test case, at least).

**Comment by harrisoncramer** [comment, marked as answer]
This is a bug, being addressed by https://github.com/harrisoncramer/gitlab.nvim/pull/225
